app/build.gradle

```diff
@@ -5,9 +5,19 @@ plugins {
     alias(libs.plugins.kotlin.parcelize)
 }
 
-final def gitSha = providers.exec {
-    commandLine('git', 'rev-parse', '--short=7', 'HEAD')
```

Comment:
By the way, why did we shorten the hash from 8 to 7 characters?

Reply:
No idea, feel free to switch back to 8